### DISCUSSION TITLE
Experimenting on multi select

### DISCUSSION THREAD

**Discussion by cloud-walker** [chakra-ui/chakra-ui#3282] [Show and tell]
I'm experimenting with Chakra, trying to recreate the "Tagging" thing of Selectize: https://selectize.github.io/selectize.js/

I'm currently focused on solving the non popover part, and with my fourth iteration I'm close to what my designer wants: https://codesandbox.io/s/multi-selects-5tixp?file=/src/modules/App/Exp3.tsx

I think I only need to find a way to propagate the  original `Input` focus style to `_focusWithin`, but I don't know how to do it.

I'll appreciate any suggestions about it!

Thanks in advance!

**Comment by csandman** [comment]
I know this isn't exactly an answer to your question, but I have recreated a similar alternative to this by wrapping `react-select` and replacing the internal components with Chakra component:

![image](https://user-images.githubusercontent.com/9214195/149229522-3240fea5-13db-4ac7-a3e7-205321090d59.png)

You can check it out here if you need something like this: https://github.com/csandman/chakra-react-select

**Comment by anubra266** [comment]
@cloud-walker, you can get the default focus style from the default theme

```ts
const inputStyles = useMultiStyleConfig("Input", {}).field as InputProps;
const focusStyles = inputStyles._focus;
  ```
